gpio/python/gpioPython/docker-compose.yml

```diff
@@ -8,13 +8,9 @@ services:
     ports:
       - 6502:6502
       - 6512:6512
-    devices:
```

Comment:
I think it is better to remove the changes on `docker-compose.yml` from this commit, and make them in the other commit